src/native/third_party/isa-l.gyp

```diff
@@ -245,14 +234,15 @@
                 'isa-l_crypto/sha256_mb/sha256_mb_x16_avx512.asm',
                 'isa-l_crypto/sha256_mb/sha256_opt_x1.asm',
                 'isa-l_crypto/sha256_mb/sha256_ni_x1.asm',
+                'isa-l_crypto/sha256_mb/sha256_ni_x2.asm',
                 'isa-l_crypto/sha256_mb/sha256_ctx_sse_ni.c',
                 'isa-l_crypto/sha256_mb/sha256_ctx_avx512_ni.c',
                 'isa-l_crypto/sha256_mb/sha256_mb_mgr_submit_sse_ni.asm',
                 'isa-l_crypto/sha256_mb/sha256_mb_mgr_flush_sse_ni.asm',
                 'isa-l_crypto/sha256_mb/sha256_mb_mgr_flush_avx512_ni.asm',
-            ],
+            ]}]],
```

Comment:
If I understand the sha1_mb/Makefile.am correctly for non x64 cases, I think that when they set `lsrc_base_aliases += sha1_mb/sha1_ctx_base_aliases.c` and then in the top Makefile.am it uses `if CPU_UNDEFINED libisal_crypto_la_SOURCES += ${lsrc_base_aliases}` that aliases is needed for the other cases. Maybe we need to add an "else" option to this condition (the third dict of the condition is "else").

```suggestion
            ]} , {
                'sources': [
                    'isa-l_crypto/sha1_mb/sha1_ctx_base_aliases.c',
                ]
            }]],
```